packages/dashboard-v2/src/services/skynetClient.js

```diff
@@ -1,3 +1,6 @@
 import { SkynetClient } from "skynet-js";
 
-export default new SkynetClient(`https://${process.env.GATSBY_PORTAL_DOMAIN}`);
+const { NODE_ENV, GATSBY_PORTAL_DOMAIN } = process.env;
+
+// In production-like environment, let SkynetClient figure out the best portal
+export default new SkynetClient(NODE_ENV === "development" ? `https://${GATSBY_PORTAL_DOMAIN}` : undefined);
```

Comment:
I don't think it works from account.domain.tld context, that'd only work for skapps hosted as skylinks because skynet-js detects the skynet-portal-api header and that is not attached for account.domain.tld responses. I think using `https://${GATSBY_PORTAL_DOMAIN}` was the right solution here - for misconfigured dev cluster it might have seem like it's working wrong but for prod portals that's expected behavior (switching between servers).

Reply:
Oh, okay -- that's a valuable piece of information, thanks :)
Reverted that change in [7248147](https://github.com/SkynetLabs/skynet-webportal/pull/2008/commits/7248147ba72f127f6f37094dbad1404d53669c65)